examples/with-http2/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "with-http2",
+  "version": "1.0.0",
+  "description": "How to run an HTTP2 server with Next.js",
+  "main": "server.js",
+  "scripts": {
+    "dev": "node server.js",
+    "build": "next build",
+    "start": "NODE_ENV=production node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zeit/next.js/"
+  },
+  "keywords": [
+    "http2",
+    "next"
+  ],
+  "author": "Shahzeb Khan <https://github.com/shahzeb1>",
```

Comment:
Can you make sure the package.json matches other examples? We generally leave out the author, repository, add in a license etc.